schemas/com.slack.sauna.commands/send_message/jsonschema/1-0-0

```diff
@@ -0,0 +1,123 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Schema for a Slack send message command",
+    "self": {
+        "vendor": "com.slack.sauna.commands",
+        "name": "send_message",
+        "format": "jsonschema",
+        "version": "1-0-0"
+    },
+    "type": "object",
+    "properties": {
+        "text": {
+            "type": "string"
+        },
+        "username": {
+            "type": "string"
+        },
+        "channel": {
+            "type": "string"
+        },
+        "icon_url": {
+            "type": "string"
+        },
+        "icon_emoji": {
+            "type": "string"
+        },
+        "link_names": {
+            "type": "boolean"
+        },
+        "mrkdwn": {
+            "type": "boolean"
+        },
+        "attachments": {
```

Comment:
Based on https://api.slack.com/docs/message-attachments#attachment_structure. Should be comprehensive.